static/js/utils.js

```diff
@@ -0,0 +1,4 @@
+// Get the HubSpot tracking cookie from the browser to be included in the form submission data.
+export function getHubspotCookie() {
+  return document.cookie.split("; ").find((row) => row.startsWith("hubspotutk="))?.split("=")[1];
```

Comment:
I used what's in the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie#example_2_get_a_sample_cookie_named_test2) to get the cookie value.